dozuki.js

```diff
@@ -3,21 +3,24 @@
    function Dozuki(url, http) {
       baseUrl = url + "/api/2.0/";
       this.guides = {
-         get: function(guideid, langid, params) {
+         get: function(guideid, langid, params, headers) {
+            headers = headers || {};
```

Comment:
Since we'll need headers on every endpoint, I'd rather have them taken in through the constructor, then always merged and sent along with each request.